plugin.json

```
@@ -35,6 +35,12 @@
                 "type": "bool",
                 "help_text": "When true, Matterpoll will render poll posts with a rich UI. The rich UI is not available on the mobile app.",
                 "default": false
+            },
+            {
+                "key": "default_settings",
+                "display_name": "Default Settings",
+                "type": "custom",
+                "help_text": "Settings will be pre-selected in 'Create Poll' dialog. Settings will be not applied to `\\poll` command."
```

```suggestion
                "help_text": "Settings will be pre-selected in 'Create Poll' dialog. Settings will not be applied to `/poll` command."
```

fix this wording again.